index.html

```diff
@@ -58,57 +51,161 @@
+                    <h1>Only For</h1>
+                    <div class="money_block">
+                      <p>$199</p>
+                      <p><del>$398</del></p>
+                    </div>
+                  </div>
+                </div>
+              </div>
             </div>
-            <div class="frame_text">
-              <i class="fa-solid fa-square-check"></i>
-              <p>Custom health pln builder</p>
+            <div class="col-xl-6">
+              <div class="hero_img">
+                <img
+                  class="hero_img"
+                  src="./public/images/heroIMG.png"
+                  alt="scientist"
+                />
+              </div>
             </div>
           </div>
-          <div class="hero_button-price">
-            <div class="button_block">
-              <button class="submit_button">
-                <a href="https://upacjenta.pl/domowe-badania-krwi">order now</a
-                ><i class="fa-solid fa-circle-chevron-right"></i>
-              </button>
+        </div>
+      </section>
+
+      <section id="dna_info">
+        <div class="dna_info_container">
+          <div class="row">
+            <div class="info_text col-xl-6 col-lg-6 col-md-10 col-sm-10">
+              <h1>What is DNA?</h1>
+              <p>
+                DNA (deoxyribonucleic acid) contains your genetic code, and it's
+                unique. You get a copy of your genes from each of your parents.
+                The places where your genes are different are called genetic
+                variants.
+              </p>
+              <p>
+                Sometimes you end up with a variation in your genetic code that
+                is associated with a health issue. Fortunately, you can change
+                the expression of many of your genes.
+              </p>
             </div>
-            <div class="price_block">
-              <h1>Only For</h1>
-              <div class="money_block">
-                <p>$199</p>
-                <p><del>$398</del></p>
+            <div class="col-xl-6 col-lg-5 col-md-12 col-sm-12">
+              <div class="movie">
+                <video controls width="250">
+                  <source src="./public/movies/DNAVideo.mp4" type="video/mp4" />
+                </video>
               </div>
             </div>
           </div>
         </div>
-        <div class="hero_img">
-          <img src="./public/images/heroIMG.png" alt="scientist" />
+      </section>
+
+      <section id="analyze">
+        <header>
+          <h1>You spit. We analyze.</h1>
+          <p>
+            Your DNA influences your health, so what does that mean for you?
+            Find out with personalized reports covering topics like:
+          </p>
+        </header>
+
+        <div class="analyze_components">
+          <div class="specrow">
+            <div class="img_container col-xl-6 col-lg-12 col-md-12 col-sm-11">
+              <div class="analyze_img">
```

Comment:
analyze_img -> analyze_image
stop using shourtcuts, especially wheere you use similar shortcuts two html language elements.